diff --git a/pyproject.toml b/pyproject.toml
index ec06143..45d86a9 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -31,7 +31,6 @@ dependencies = [
     "scipy",
     "scikit-learn",
     "numba>=0.57",
-    "stochastic",
 ]
 
 [project.optional-dependencies]
@@ -48,6 +47,7 @@ docs = [
     "numpydoc",
     "sphinx-copybutton",
     "sphinx-design",
+    "stochastic",  # for doctests
     # "sphinx-notfound-page",
 ]
 
